app/src/main/java/com/zulip/android/activities/RecyclerMessageAdapter.java

```diff
@@ -381,10 +382,11 @@ public void onBindViewHolder(final RecyclerView.ViewHolder holder, final int pos
                             .setOnClickListener(new View.OnClickListener() {
                                 @Override
                                 public void onClick(View view) {
-                                    Intent i = new Intent(Intent.ACTION_VIEW);
+                                    Intent i = new Intent(zulipApp.getApplicationContext(), PhotoViewActivity.class);
                                     i.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
-                                    i.setData(Uri.parse(url));
+                                    i.putExtra(Intent.EXTRA_TEXT , url);
                                     zulipApp.startActivity(i);
+                                    ((Activity) context).overridePendingTransition(android.R.anim.fade_in, android.R.anim.fade_out);
```

Comment:
@kunall17  `android.R.anim.fade_in` and  `android.R.anim.fade_out` is best for now i guess.  required then we can do it in future. :)